DuckDuckGo/TabViewController.swift

```diff
@@ -852,15 +850,6 @@ extension TabViewController: WKNavigationDelegate {
             SKStoreReviewController.requestReview()
             appRatingPrompt.shown()
         }
-        
-        // If site is unprotected we need to remove the content blocking rules
-        if let ruleList = ContentBlockerRulesManager.shared.blockingRules {
-            if !contentBlockerProtection.isProtected(domain: url?.host) {
-                webView.configuration.userContentController.remove(ruleList)
-            } else {
-                webView.configuration.userContentController.add(ruleList)
-            }
-        }
```

Comment:
@SlayterDev Does this look good to you? The rules themselves will be generated with exclusions for protected domains so IMO this code is fine to take out, but want to get your take. 🙂